packages/cspell-lib/src/Settings/DefaultSettings.ts

```diff
@@ -61,27 +61,8 @@ type PredefinedPatternNames = ExtendsType<NameType<typeof predefinedPatterns>, P
 
 const defaultRegExpPatterns: RegExpPatternDefinition[] = [...predefinedPatterns].map(normalizePattern);
 
-const definedDefaultRegExpExcludeList: PredefinedPatterns[] = [
```

Comment:
I moved it out because I assumed that the doc generator would not generate docs for non-exported variables.

If this was a wrong assumption, then let me know and I will move it back.